index.html

```diff
@@ -1,16 +1,24 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <meta charset="utf-8">
-  <title></title>
+  <link rel="stylesheet" href="base.css">
+  <link rel="stylesheet" href="module.css">
+  <!--Need to link all three CSS files?-->
+  <!-- Correct place for link tags? -->
+  <!-- Need rel attribute? -->
 </head>
 <body>
+  <header>
+  </header>
+  <main>
   <section id="projects">
-    <a href="project source">Title</a>
-    <p class="date-created"></p>
-    <img src="relevant image">
+    <article class="template">
+      <a href="project source">Project Title</a>
+      <p class="date-created"></p>
```

Comment:
Depending on how you want your projects to display on the page, you might consider making your dateCreated property show up as text inside of your paragraph instead of as an attribute.